include/PPN-microbench/ops.hpp

```diff
@@ -18,8 +17,8 @@ class Ops : public Microbench {
 
     size_t results[6][RUNS];
 
-    template <class T> void benchhaha(T *);
-    template <class T> void benchSIMD(T *);
+    // 'reps' represents the number of operations executed in 'f'
+    template <class T> void wrap(T (*f)(T, T), T *val, int reps);
```

Comment:
What is this `wrap` function?
Careful! Too much abstraction is not a good thing either.